gpu.Dockerfile

```diff
@@ -44,6 +44,13 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 ENV CUDA_HOME=/usr/local/cuda
 
+# Hack to fix R trying to use CUDA in `/usr/lib/x86_64-linux-gnu` directory instead
+# of `/usr/local/nvidia/lib64` (b/152401083).
+# For some reason, the CUDA file `libcuda.so.418.67` in the former directory is empty.
+# R's ldpaths modifies LD_LIBRARY_PATH on start by adding `/usr/lib/x86_64-linux-gnu` upfront.
+# Instead, this version of ldpaths adds it at the end.
+ADD ldpaths $R_HOME/etc/ldpaths
```

Comment:
Are there libraries present in `/usr/lib/x86_64-linux-gnu` but not elsewhere?

Reply:
The cuda libraries? They are available in another place:
```
root@8ba51740576e:/# find /usr -name libcuda.so.418.67
/usr/lib/x86_64-linux-gnu/libcuda.so.418.67
/usr/local/nvidia/lib64/libcuda.so.418.67
```

Reply:
`/usr/lib/x86_64-linux-gnu` includes 2410 libraries. If we remove them from the path, are we sure they are available at some other places?

```
ls /usr/lib/x86_64-linux-gnu | wc -l
2410
```

Reply:
I didn't remove it from the path, just put it at the end instead of the beginning.

Added the original file in the PR comment if you want to compare.